bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -737,6 +738,68 @@ impl Room {
         self.inner.send_queue().send(replacement_event).await?;
         Ok(())
     }
+
+    /// Remove verification requirements for the given users and
+    /// resend messages that failed to send because their identities were no
+    /// longer verified (in response to
+    /// `SessionRecipientCollectionError::VerifiedUserChangedIdentity`)
+    pub async fn withdraw_verification_and_resend(
+        &self,
+        user_ids: Vec<String>,
+        transaction_id: String,
+    ) -> Result<(), ClientError> {
+        let transaction_id: OwnedTransactionId = transaction_id.into();
+
+        let parsing_result: Result<Vec<OwnedUserId>, _> =
+            user_ids.iter().map(UserId::parse).collect();
```

Comment:
One of these should work:

```suggestion
        let user_ids: Vec<OwnedUserId> = user_ids.iter().map(UserId::parse).collect()?;
```


```suggestion
        let user_ids: Vec<OwnedUserId> = user_ids.iter().map(UserId::parse).collect::<Result<_, _>>()?;
```

Reply:
Oh sweet, the second one does! So I can specialise collect to avoid inference issues, that's the bit I was missing when I first tried to write it that way. Really cool 👍